node/plugin/cmd/main.go

```diff
@@ -135,12 +136,36 @@ func pluginOps(ctx *cli.Context) {
 		return
 	}
 
-	_, dispersalPort, retrievalPort, v2DispersalPort, v2RetrievalPort, err := core.ParseOperatorSocket(config.Socket)
+	opSocket, err := core.ParseOperatorSocket(config.Socket)
 	if err != nil {
 		log.Printf("Error: failed to parse operator socket: %v", err)
 		return
 	}
 
+	_, dispersalPort, err := net.SplitHostPort(opSocket.GetV1DispersalSocket())
+	if err != nil {
+		log.Printf("Error: failed to get dispersal port from the socket: %s, err: %s", opSocket.Normalize(), err)
```

Comment:
It may be better to log the `config.Socket`, let users see which param they supplied is not right

Reply:
Ack. Good call.

Reply:
Parsing errors are logged on line 141